io.github.yesser_studios.Pong.yml

```diff
@@ -0,0 +1,51 @@
+id: io.github.yesser_studios.Pong
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+finish-args:
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=host-os:ro
```

Comment:
why do you need that extremely broad filesystem access?

Reply:
I put it in there for testing, I'll remove it